core/tests/protractor_utils/forms.js

```diff
@@ -461,11 +486,11 @@ var expectRichText = function(elem) {
         // applying .getText() while the RichTextChecker is running would be
         // asynchronous and so not allow us to update the textPointer
         // synchronously.
-        return await entry.getText();
+        return await action.getText('Entry', entry);
```

Comment:
Do we know what `entry` is? Can we get a more descriptive name for the first argument?

Reply:
done!